library/std/src/sys/backtrace.rs

```diff
@@ -126,11 +135,15 @@ unsafe fn _print_fmt(fmt: &mut fmt::Formatter<'_>, print_fmt: PrintFmt) -> fmt::
             }
 
             idx += 1;
+            last_sp = frame.sp();
```

Comment:
What is the behaviour of `sp()` when the frame pointer is omitted? Is it guaranteed to be null? Because otherwise, this lead to a false-positive as `last_sp` is set even for the first frame.

Reply:
Frames 0 and 1 are skipped by the `idx > 1` condition. And after that even with frame pointers omitted the stack pointer has to change to store the return pointer.

Reply:
Whoops, I misread `idx > 1` as `idx > 0`, so the check is only run after `last_sp` has definitely been set to a valid value. Nevermind...